apps/shell/Kconfig

```diff
@@ -59,18 +59,35 @@ config TASH_SCRIPT
 
 config SECURED_TASH
 	bool "Enable Secure shell"
-	depends on NET_SECURITY_TLS
+	depends on NET_SECURITY_TLS && ENABLE_IOTIVITY && ENABLE_IOTIVITY_SECURED
```

Comment:
I understand to have a dependency with TLS, but don't know why it has iotivity dependencies. Could you check it?

Reply:
`MBEDTLS_PKCS5_C` should be enabled to use `mbedtls_pkcs5_pbkdf2_hmac`.
But `MBEDTLS_PKCS5_C` is related to `MBEDTLS_OCF_PATCH`. 

```
#undef MBEDTLS_PKCS5_C
#if defined(MBEDTLS_OCF_PATCH)
#define MBEDTLS_PKCS5_C
#endif
```

MBEDTLS_OCF_PATCH is enabled when enables IOTIVITY_SECURED. That's why I add dependenciy in Iotivity.